app/views/campaigns/overview.html.haml

```diff
@@ -1,4 +1,8 @@
+- content_for :before_title, "#{@campaign.title} - "
+- content_for :after_title, ' - Overview'
+
```

Comment:
Instead of it being "Spring 2015 - Wiki Education Foundation Dashboard - Programs"(looks disconnected) Make it something like Spring 2015  Programs - Wiki Education Foundation Dashboard